Tests/scripts/infrastructure_tests/mock_unit_tests.py

```diff
@@ -0,0 +1,31 @@
+from mock import patch
+from Tests.mock_server import AMIConnection, clean_filename, get_mock_file_path, get_log_file_path, get_folder_path
+
+
+def test_clean_filename():
+    assert clean_filename(u'th)))i(s) is a (test8)8   8') == 'th___i_s__is_a__test8_8___8'
+    assert clean_filename(u'n&%ew $r#eplac@es', replace='&%$#@') == 'n__ew _r_eplac_es'
+
+
+def test_silence_output():
+    # TODO: How to check output is redirected to /dev/null ?
+    pass
+
+
+def test_get_paths():
+    test_playbook_id = u'test_playbook'
+    assert get_mock_file_path(test_playbook_id) == 'test_playbook/test_playbook.mock'
+    assert get_log_file_path(test_playbook_id) == 'test_playbook/test_playbook_playback.log'
+    assert get_log_file_path(test_playbook_id, record=True) == 'test_playbook/test_playbook_record.log'
+    assert get_folder_path(test_playbook_id) == 'test_playbook/'
+
+
+# TODO: Maybe mock subprocess functions??
```

Comment:
I think it should be in a function

Reply:
This will become a test set that will mock many functions of the "subprocess" module, and then run many tests.